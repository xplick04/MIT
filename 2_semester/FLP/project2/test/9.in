A D
A E
A F
B D
B E
B F
C D
C E
C F
C G
