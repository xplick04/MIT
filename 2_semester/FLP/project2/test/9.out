A-D A-E A-F B-D C-D C-G
A-D A-E A-F B-D C-E C-G
A-D A-E A-F B-D C-F C-G
A-D A-E A-F B-E C-D C-G
A-D A-E A-F B-E C-E C-G
A-D A-E A-F B-E C-F C-G
A-D A-E A-F B-F C-D C-G
A-D A-E A-F B-F C-E C-G
A-D A-E A-F B-F C-F C-G
A-D A-E B-D B-F C-D C-G
A-D A-E B-D B-F C-E C-G
A-D A-E B-D B-F C-F C-G
A-D A-E B-D C-D C-F C-G
A-D A-E B-D C-E C-F C-G
A-D A-E B-E B-F C-D C-G
A-D A-E B-E B-F C-E C-G
A-D A-E B-E B-F C-F C-G
A-D A-E B-E C-D C-F C-G
A-D A-E B-E C-E C-F C-G
A-D A-E B-F C-D C-F C-G
A-D A-E B-F C-E C-F C-G
A-D A-F B-D B-E C-D C-G
A-D A-F B-D B-E C-E C-G
A-D A-F B-D B-E C-F C-G
A-D A-F B-D C-D C-E C-G
A-D A-F B-D C-E C-F C-G
A-D A-F B-E B-F C-D C-G
A-D A-F B-E B-F C-E C-G
A-D A-F B-E B-F C-F C-G
A-D A-F B-E C-D C-E C-G
A-D A-F B-E C-E C-F C-G
A-D A-F B-F C-D C-E C-G
A-D A-F B-F C-E C-F C-G
A-D B-D B-E B-F C-D C-G
A-D B-D B-E B-F C-E C-G
A-D B-D B-E B-F C-F C-G
A-D B-D B-E C-D C-F C-G
A-D B-D B-E C-E C-F C-G
A-D B-D B-F C-D C-E C-G
A-D B-D B-F C-E C-F C-G
A-D B-D C-D C-E C-F C-G
A-D B-E B-F C-D C-E C-G
A-D B-E B-F C-D C-F C-G
A-D B-E C-D C-E C-F C-G
A-D B-F C-D C-E C-F C-G
A-E A-F B-D B-E C-D C-G
A-E A-F B-D B-E C-E C-G
A-E A-F B-D B-E C-F C-G
A-E A-F B-D B-F C-D C-G
A-E A-F B-D B-F C-E C-G
A-E A-F B-D B-F C-F C-G
A-E A-F B-D C-D C-E C-G
A-E A-F B-D C-D C-F C-G
A-E A-F B-E C-D C-E C-G
A-E A-F B-E C-D C-F C-G
A-E A-F B-F C-D C-E C-G
A-E A-F B-F C-D C-F C-G
A-E B-D B-E B-F C-D C-G
A-E B-D B-E B-F C-E C-G
A-E B-D B-E B-F C-F C-G
A-E B-D B-E C-D C-F C-G
A-E B-D B-E C-E C-F C-G
A-E B-D B-F C-D C-E C-G
A-E B-D B-F C-E C-F C-G
A-E B-D C-D C-E C-F C-G
A-E B-E B-F C-D C-E C-G
A-E B-E B-F C-D C-F C-G
A-E B-E C-D C-E C-F C-G
A-E B-F C-D C-E C-F C-G
A-F B-D B-E B-F C-D C-G
A-F B-D B-E B-F C-E C-G
A-F B-D B-E B-F C-F C-G
A-F B-D B-E C-D C-F C-G
A-F B-D B-E C-E C-F C-G
A-F B-D B-F C-D C-E C-G
A-F B-D B-F C-E C-F C-G
A-F B-D C-D C-E C-F C-G
A-F B-E B-F C-D C-E C-G
A-F B-E B-F C-D C-F C-G
A-F B-E C-D C-E C-F C-G
A-F B-F C-D C-E C-F C-G
